commerce-cloud-code/app_yourshophere/cartridge/partials/cart/addtocart.js

```diff
@@ -15,6 +15,13 @@ exports.createModel = function createModel(options) {
         price: StringUtils.formatMoney(lineitem.price),
         cartUrl: URLUtils.url('Cart-Show'),
         itemCount: options.basket.productQuantityTotal,
+        analytics: JSON.stringify({
```

Comment:
I am thinking to add a dedicated scriptModule for this.

`require('*/tracking/cart').lineItemAnalytics(lineitem);`

That'd encourage folks to extend our concept vs workarounding it with non performant approaches